test/release/examples/primers/modules.chpl

```diff
@@ -150,6 +150,36 @@ module MainModule {
     // var twiceFoo = 2 * foo;
 
 
+    /* An ``import`` statement, in contrast to a ``use`` statement, either
+       enables qualified access to the visible symbols of a module or enables
+       unqualified access to a specified subset of those symbols.
+
+       This demonstrates an ``import`` that enables qualified access.  In this
+       example, ``bazBarFoo`` should store the result of calling
+       ``modToUse.baz`` on ``modToUse.bar`` and ``modToUse.foo``, which is in
+       this case ``28``.
+    */
+    {
+      import modToUse;
+
+      var bazBarFoo = modToUse.baz(modToUse.bar, modToUse.foo);
+      writeln(bazBarFoo);
+    }
+    /* This case demonstrates an ``import`` that enables unqualified access to
+       a single symbol within ``modToUse``.
+    */
```

Comment:
I am slightly worried that "qualified" and "unqualified" access might be a bit too technical for a primer, at least without an example first. I see some parenthesized explanation above, but I am not sure if it is enough.

Maybe the paragraph above can be changed so that it says something like:

"This demonstrates a usage of an ``import`` statement.
.... 
Accesses to symbols via explicitly naming the module that they are defined in are 'qualified' accesses"

I am not thinking too strongly about this, though